### PR TITLE
Fixup build and docs after move to standalone repo

### PR DESCRIPTION
**Pull request by dibenede**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -153,54 +159,52 @@ function getClosureCompilerCommand(exportsFile, outputFile) {
   ].join(' ');
 }
 
-gulp.task('dist', gulp.series(['genproto_wellknowntypes'], function(cb) {
+function gen_google_protobuf_js(cb) {
   // TODO(haberman): minify this more aggressively.
   // Will require proper externs/exports.
+  var log_file = fs.createWriteStream('/usr/local/google/home/dibenede/gulp.log', {flags:'w'});
```
[the remark this answers is not in the document]
Comment (dibenede) [reply gulpfile.js:165]: blah , thanks. that was from my debugging attempts. removed wholesale.

File: gulpfile.js [hunk 2]
```diff
@@ -153,54 +159,52 @@ function getClosureCompilerCommand(exportsFile, outputFile) {
...
+            exec(
+                'mkdir -p commonjs_out/test_node_modules && ' +
+                  getClosureCompilerCommand(
+                      'commonjs/export_testdeps.js',
+                      'commonjs_out/test_node_modules/testdeps_commonjs.js'),
+                function(err, stdout, stderr) {
+                  console.log(stdout);
+                  console.log(stderr);
+                  cb(err);
+                });
```
[the remark this answers is not in the document]
Comment (dibenede) [reply gulpfile.js:204]: done

File: gulpfile.js [hunk 3]
```diff
@@ -153,54 +159,52 @@ function getClosureCompilerCommand(exportsFile, outputFile) {
...
+
+function commonjs_testdeps(cb) {
+            exec(
+                'mkdir -p commonjs_out/test_node_modules && ' +
+                  getClosureCompilerCommand(
+                      'commonjs/export_testdeps.js',
+                      'commonjs_out/test_node_modules/testdeps_commonjs.js'),
+                function(err, stdout, stderr) {
+                  console.log(stdout);
+                  console.log(stderr);
```
[the remark this answers is not in the document]
Comment (dibenede) [reply gulpfile.js:202]: done

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "google-protobuf",
+  "name": "google-protobuf-javascript",
```
[the remark this answers is not in the document]
Comment (dibenede) [reply package.json:2]: done

File: gulpfile.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukesandberg) [reply gulpfile.js:137]: not done

[the remark this answers is not in the document]
Comment (dibenede) [reply gulpfile.js]: done -er

**Review by lukesandberg — APPROVED** [review]

**Pull request merged**